src/pages/questionPage.js

```diff
@@ -18,10 +21,58 @@ export const initQuestionPage = () => {
   userInterface.appendChild(questionElement);
 
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
+  answersListElement.classList.add('answers-list');
+
+  // This block displays current score
+  const currentScores = document.createElement('div');
+  currentScores.classList.add('current-scores');
+  currentScores.innerHTML = `Correct answers: ${correctCounter} of ${quizData.questions.length}`;
+  userInterface.appendChild(currentScores);
 
   for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
     const answerElement = createAnswerElement(key, answerText);
+    // Answers options look little bit better now
+    answerElement.classList.add('answer-element');
     answersListElement.appendChild(answerElement);
+
+    answerElement.addEventListener('click', function () {
+      // When you click the answer, previous selection loses selection color
+      const answerElements = document.querySelectorAll('.answer-element');
+      answerElements.forEach((answer) => {
+        answer.classList.remove('answer-selected');
+        answer.classList.remove('answer-correct');
+        answer.classList.remove('answer-incorrect');
```

Comment:
You can remove multiple classes by passing in more strings to the same `remove` call.